setup.py

```diff
@@ -17,7 +17,7 @@
         'apache-libcloud',
         'docopt==0.6.2',
         'gevent==1.4.0',
-        'reportportal-client<4.0.0',
+        'reportportal-client==3.2.3',
```

Comment:
Are we limited here due to the version of report portal running on cistatus?

Reply:
@clacroix12 that is correct, the newer version has moved to pgsql and we are at 4.x